config.yaml

```
@@ -18,6 +18,36 @@ options:
       kubeapi-load-balancer charm. The workers will evenly distribute the load by alternating
       between the IP addresses in this list. For example, if you have 2 IPs and 4 workers,
       each IP will be used by 2 workers.
+  events-context:
+    type: string
+    default: ""
+    description: |
+      This field allow to include directives for the events context in the Nginx Configuration file.
+      It should be formatted as a YAML string. For instance:
```

`yaml` config is certainly tricky to get right -- either in a bundle or from the command line.   But i see where you're going

```suggestion
      This option allows the juju operator to configure directives inside the nginx events context in /etc/nginx/nginx.conf. Provide a valid block of YAML. For instance:
```